### PR TITLE
feat: Adding some extra config to `config.sh` script

### PR DESCRIPTION
**Pull request by jcortejoso**
Adding some additional exposure to `config.sh` script to expose on `.envrc` configurations that may be interesting to consider when deploying a new testnet.
Also included a small fix on Makefile caused by a dead npm script: https://github.com/ethereum-optimism/optimism/pull/10821

### CODE REVIEW

**Comment by jcortejoso** [comment]
Guide I used for deploying from this branch:
https://gist.github.com/jcortejoso/55baa06eb309199b84219f98d4817a67

File: packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh [hunk 1]
```diff
@@ -101,11 +111,18 @@ config=$(cat << EOL
...
   "disputeGameFinalityDelaySeconds": 302400,
-  "respectedGameType": 0
+  "respectedGameType": 0,
+
+  "usePlasma": $USE_PLASMA,
+  "daCommitmentType": "KeccakCommitment",
+  "daChallengeWindow": 160,
+  "daResolveWindow": 160,
+  "daBondSize": 1000000,
+  "daResolverRefundPercentage": 0
```
Comment (palango) [inline packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh:129]: I'm not sure if there's a use case for it, but you could add `deployCeloContracts` here as well.

**Review by palango — APPROVED** [review]
Left one optional remark, looks good otherwise.

File: packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh [hunk 2]
```diff
@@ -65,10 +74,12 @@ config=$(cat << EOL
   "l1FeeVaultWithdrawalNetwork": 0,
   "sequencerFeeVaultWithdrawalNetwork": 0,
 
-  "gasPriceOracleOverhead": 2100,
+  "gasPriceOracleOverhead": 0,
   "gasPriceOracleScalar": 1000000,
 
-  "enableGovernance": true,
+  "deployCeloContracts": true,
```
Comment (palango) [inline packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh]: Is this to use together with migrated state? Then it should default to `false`.

Comment (jcortejoso) [reply packages/contracts-bedrock/scripts/getting-started/config-vars-celo.sh]: Good to know, I'll add as a `.envrc` variable then

**Review by palango — APPROVED** [review]

**Pull request merged**